flutter/lib/backend/bridge/isolate.dart

```diff
@@ -48,7 +49,27 @@ class BridgeIsolate {
     sendPort.send(port.sendPort);
     await for (var settings in port.cast<RunSettings>()) {
       var result = runBenchmark(settings);
+      result.validity = await isResultValid(settings);
       sendPort.send(result);
     }
   }
+
+  static Future<bool> isResultValid(RunSettings settings) async {
```

Comment:
Can you add a comment or relevant link to explain why we use this logic to check for validity? In case the logic is changed by external party we follow and can update it.

Reply:
Done